snapyr/src/main/java/com/snapyr/sdk/Snapyr.java

```diff
@@ -214,8 +214,8 @@ public void handleMessage(Message msg) {
             this.sendQueue =
                     new BatchUploadQueue(application, 999999, 999, new BatchQueue.MemoryQueue());
         } else {
-        this.sendQueue =
-                new BatchUploadQueue(application, flushIntervalInMillis, flushQueueSize, null);
+            this.sendQueue =
```

Comment:
Favorite change of the year

Reply:
Thank you... spotless worked hard to tweak all the spaces